frontend/src/lib/components/details/recommendations.svelte

```diff
@@ -11,31 +12,42 @@
   export let mediaType: string
 
   let loadedPage: boolean
-  let slidesPerView: number
 
-  // TODO: Make this happen in a less insane way
   onMount(() => {
-    slidesPerView = calculateAmountOfShownItems({
-      width: window.visualViewport.width,
-      xxl: 7,
-      xl: 6,
-      lg: 5,
-      md: 4,
-      sm: 3,
-      mobile: 2,
-    })
     loadedPage = true
   })
 </script>
 
 {#if recommendations.length && loadedPage}
   <h1 class="text-center text-3xl pt-5 pb-5">Recommendations</h1>
-  <div class="bg-neutral rounded-box swiper-container px-2">
+  <div class="bg-neutral rounded-box swiper-container px-2 mx-2">
     <Swiper
+      style="
+        --swiper-navigation-color: text-blue-500;
+        --swiper-navigation-size: 25px;
+      "
+      grabCursor={true}
+      resistance={false}
+      preloadImages={false}
+      lazy={{
+        enabled: true,
+        checkInView: true,
+        loadPrevNext: true,
+      }}
       spaceBetween={15}
-      {slidesPerView}
+      watchSlidesProgress={true}
+      breakpoints={{
+        0: { slidesPerView: 2 },
+        576: { slidesPerView: 3 },
+        768: { slidesPerView: 4 },
+        992: { slidesPerView: 5 },
+        1200: { slidesPerView: 6 },
+        1400: { slidesPerView: 7 },
+      }}
```

Comment:
Dno where to write this, but the recommendations seems to be squished 
![image](https://user-images.githubusercontent.com/43907402/202476155-16a41e7c-6e51-450a-abd5-995b22889c51.png)